addons/stock_account/views/stock_valuation_layer_views.xml

```diff
@@ -48,7 +48,7 @@
                 <field name="quantity" />
```

Comment:
same things on quantity can be useful

`<field name="quantity" sum="Total Quantity"/>`

Reply:
not really since you can have different UOM categories.

Reply:
@nim-odoo you are right

(Note : it is the same issue if the currency is not the same. this case is less common).